src/responder/common/negcache.c

```diff
@@ -63,14 +66,50 @@ static int string_to_tdb_data(char *str, TDB_DATA *ret)
     return EOK;
 }
 
+static errno_t ncache_load_nss_symbols(struct sss_nss_ops *ops)
+{
+    errno_t ret;
+    size_t i;
+
+    ret = sss_load_nss_symbols(ops, "files");
+    if (ret != EOK)
```

Comment:
Please put the opening `{` at the same line as the if

Reply:
Fixed.